package.json

```diff
@@ -11,6 +11,7 @@
     "debug": "^2.1.1",
     "denodeify": "^1.2.0",
     "es6-promise": "^2.0.1",
+    "fastly": "^1.2.1",
```

Comment:
think i might instigate a one in one out policy for dependencies…… don't want an Origami Build Tools 2 minute installation……